lifemonitor/api/services.py

```diff
@@ -28,7 +28,10 @@ def __init__(self):
 
     @classmethod
     def _find_and_check_workflow(cls, uuid, version, user: User):
```

Comment:
All methods in this class are declared as `@classmethod`. However, only a few of them actually use the `cls` first arg. All the others can be declared as `@staticmethod`, omitting the `cls` arg. E.g.:
```python
    @staticmethod
    def _find_and_check_workflow(uuid, version, user: User):
```

Reply:
Right! Fixed by https://github.com/crs4/life_monitor/pull/20/commits/44ca622c8a37da315096ac96d992ae8765355eef